src/main/java/htsjdk/samtools/SamFileValidator.java

```diff
@@ -508,6 +518,10 @@ private void validateMateFields(final SAMRecord record, final long recordNumber)
         }
         validateMateCigar(record, recordNumber);
 
+        if (skipMateValidation) {
+            return;
```

Comment:
You might want a test for this.  Check that it validates the mate cigar but not missing mates.   Otherwise the monkeys tampering with the code in the future could change the semantics unexpectedly.